nexus/reconfigurator/planning/tests/output/zone_image_source_change_1.txt

```diff
@@ -0,0 +1,116 @@
+    crucible          88b48e39-9a40-406f-85e0-e2668a5bb29f   install dataset                                                                in service    fd00:1122:3344:101::27
+    crucible          90eac14e-ffe2-4275-b936-04038ad5f1da   install dataset                                                                in service    fd00:1122:3344:101::2c
+    crucible          c00545ae-87f2-4de0-855e-df3c192c3d73   install dataset                                                                in service    fd00:1122:3344:101::2b
+    crucible          e2b28776-80d3-4eb9-a051-d213201ff98b   install dataset                                                                in service    fd00:1122:3344:101::30
+    crucible          f90b1d96-5795-4c60-bdce-96945bfc97f8   install dataset                                                                in service    fd00:1122:3344:101::2a
+    crucible          fdad6b6c-4442-415b-a66d-2b292286b8b7   install dataset                                                                in service    fd00:1122:3344:101::2f
+    crucible_pantry   6547ed1c-ac1c-4ae5-a981-744ee640ba4b   install dataset                                                                in service    fd00:1122:3344:101::25
+    crucible_pantry   de412a19-e86e-442c-b951-db98e04dc732   install dataset                                                                in service    fd00:1122:3344:101::24
+    crucible_pantry   e62ecc85-209a-42f6-9c93-b5ec4f0ffcf6   install dataset                                                                in service    fd00:1122:3344:101::26
+    internal_dns      04daf9ae-4c0d-4e40-b6f8-ccf045282c51   install dataset                                                                in service    fd00:1122:3344:3::1   
+    internal_dns      9f005230-675a-464b-b4fd-adc1ad3c0e9c   install dataset                                                                in service    fd00:1122:3344:1::1   
+    internal_dns      a44ffea0-7b34-4ec9-8b6c-1d3871738c41   install dataset                                                                in service    fd00:1122:3344:2::1   
+    internal_ntp      6d192632-6f91-4eaf-8f47-ae762da6a937   install dataset                                                                in service    fd00:1122:3344:101::21
+    nexus             a75338ce-fbc4-4018-9986-ba7cf62af1d6   install dataset                                                                in service    fd00:1122:3344:101::22
+*   crucible          01fbc62f-dd37-4d23-a4f9-e678cc4210c9   - install dataset                                                              in service    fd00:1122:3344:101::28
+     └─                                                      + artifact: 1212121212121212121212121212121212121212121212121212121212121212                                       
```

Comment:
Mm, this is a specific example of what I was worried about. Do we want this to be something like `artifact: 13.0.0`?

Reply:
Changed.